inc/namespace.php

```diff
@@ -267,6 +271,22 @@ function add_login_logo() {
 	<?php
 }
 
+/**
+ * Undocumented function
+ *
+ * @param string $url Filters the site icon URL.
+ *
+ * @return string
+ */
+function filter_favicon( $url ) {
+	if ( ! $url ) {
```

Comment:
I think it's best to use `empty( $url )` instead of `! $url` to be more explicit. You could also add type hints here at least for the return value.